doc/getting-started/installation-instructions/cluster.md

```diff
@@ -84,7 +84,7 @@ NFS for example).
 On the master node you will also need space for:
 
 - the **shared_dir** directory (usually located under `/home`): it contains the calculations datastore (`hdf5` files located in the `oqdata` folder)
-- the OpenQuake database (located under `/var/lib/openquake/oqdata/`): it contains only logs and metadata, the expected size is tens of megabyte
+- the OpenQuake database (located under `/opt/openquake/venv/oqdata/`): it contains only logs and metadata, the expected size is tens of megabyte
```

Comment:
Having the database inside the venv is very wrong. It is really there or you made a mistake?

Reply:
I check again the install.py, you are right the oqdata is outside of venv